### PR TITLE
Add a primary mixin and remove IE8 support.

### PR DESCRIPTION
**Pull request by notlee**
- Removes IE8 support (sorry: merged with this PR for convenience).
- Adds a primary mixin `oGrid`.
- Removes mixins and variables replaced by `oGrid` (see MIGRATION.md).
- Removes `$o-grid-debug-mode` in favour of calling `oGridDebugInfo`.

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
 ### Migrating from v4 to v5
 
+### Sass
+
+IE8 is no longer supported, remove uses of:
+- `$o-grid-ie8-rules`
+- `oGridTargetIE8`
+- `oGridTargetModernBrowsers`
+
+The following Sass mixins and variables have been removed. Replace them with a single call to `oGrid` with the relevant options. See [the README](./README.md) for more details:
```
Comment (JakeChampion) [inline MIGRATION.md]: 👍

**Comment by notlee** [comment]
_A future PR will revisit the README, including improving the `oGrid` entry._

File: main.scss [hunk 2]
```diff
@@ -1,87 +1,150 @@
...
 @import 'sass-mq/mq';
-@import 'src/scss/main';
+@import 'src/scss/variables';
+@import 'src/scss/functions';
+@import 'src/scss/mixins';
 
-/// Output debug information about the currently loaded layouts.
+/// Output o-grid styles. This mixin outputs all styles by default.
+/// Pass an options argument to include styles granularly, see the
+/// [README](https://registry.origami.ft.com/components/o-grid/readme) for more details.
```
Comment (JakeChampion) [inline main.scss]: 👍

**Comment by JakeChampion** [comment]
Bye Bye IE 8

**Pull request merged**